src/execution_plan/ops/op_conditional_traverse.c

```diff
@@ -66,7 +72,7 @@ void _traverse(CondTraverse *op) {
 	GrB_Matrix_clear(op->F);
 }
 
-int CondTraverseToString(const OpBase *ctx, char *buff, uint buff_len) {
+static int ToString(const OpBase *ctx, char *buff, uint buff_len) {
```

Comment:
Don't you want to keep `CondTraverseToString`?

Reply:
Thanks, I missed these somehow!